cdap-security/pom.xml

```diff
@@ -172,6 +182,12 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>commons-logging</groupId>
```

Comment:
We should use jcl-over-slf4j instead of commons-logging

Reply:
Please address this comment

Reply:
I would try to disable metrics by setting dfs.namenode.metrics.logger.period.seconds to -1. This code is weird and refers log4j. Also the commons-logging would collide with jcl-over-slf4j, so logging will be all over the place

Reply:
thanks, this worked.